src/components/intro-page/requests.ts

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Why is this not in the API dir? Or should this be part of the frontend and not be delivered by the backend?

Reply:
The file is not in the API directory, because it's only part of the frontend.